iam/x509-identity-mgmt/js/schemas/defs.json

```diff
@@ -44,7 +44,7 @@
           ],
           "title": "Device ID",
           "description": "Identifies the device that the x.509 certificate is attached to.",
-          "maxLength": 10,
+          "maxLength": 36,
```

Comment:
Why 36?

Reply:
To be consistent with the regular expression `^[0-9a-fA-F-]{6,36}$` of the bottom line